packages/material-ui-docs/src/MarkdownElement/MarkdownElement.js

```diff
@@ -83,11 +83,11 @@ const styles = theme => ({
     fontSize: 16,
     color: theme.palette.text.primary,
     '& .anchor-link': {
-      marginTop: -theme.spacing.unit * 12, // Offset for the anchor.
+      marginTop: -8 * 12, // Offset for the anchor.
```

Comment:
Even though it's a bit more work, I would just make this `-96`, and so on.

Reply:
Alright